packages/wrangler/src/api/startDevWorker/BundlerController.ts

```diff
@@ -154,22 +170,21 @@ export class BundlerController extends Controller<BundlerControllerEventMap> {
 		await this.#customBuildWatcher?.close();
 		this.#customBuildAborter?.abort();
 
-		if (!config.build?.custom.command) {
+		if (!config.build?.custom?.command) {
 			return;
 		}
 
-		assert(config._entry);
-		assert(config.build?.custom.watch);
+		assert(config.build?.custom?.watch);
 
-		this.#customBuildWatcher = watch(config.build?.custom.watch, {
+		this.#customBuildWatcher = watch(config.build?.custom?.watch, {
```

Comment:
the assert above should make these nullish operators unnecessary, right? in fact, we'd never want to pass `undefined` to `watch()`